src/paths/index/components/changes/ReleaseNotesItem.js

```diff
@@ -306,6 +319,22 @@ export default class ReleaseNotesItem extends LitElement {
             <div class="item-container">
                 <div class="item-changes">
                     <div class="item-changes-types">
+                        <span
+                            class="item-changes-type"
+                            @click="${() => {
+                                this._hideBugs = !this._hideBugs;
+                                this._updateNotes();
+                                this.requestUpdate();
+                            }}"
+                        >
+                            <input
+                                type="checkbox"
+                                .checked="${this._hideBugs}"
+                                @change="${this._onBugChange}"
+                            >
+                            Hide Bugs
```

Comment:
Since this hides bug fix commits (and not known bug lines), it makes more sense to call it this way:

```suggestion
                            Hide Bug Fixes
```

Reply:
Your right, I changed it to what you suggested.